docs/quickstart/send-report-ui.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Send observability report"
+---
+
+import InstallCLI from "../snippets/install-cli.mdx";
+import InstallDbtPackage from "../snippets/install-dbt-package.mdx";
+
```

Comment:
You imported the components but didn't include them.
Refer to the 'generate-report-ui' page to see how to do it.

Reply:
I deleted them since they're not needed.

Reply:
The only dilemma is if we need to look at the page as "part of a flow" or "all you need to setup this e2e".
This is what Or mentioned this morning.
Anyway, we can decide on it later. I'm fine with this for now.